app/services/state_file/state_information_service.rb

```diff
@@ -42,6 +50,27 @@ def state_code_from_intake_class(klass)
         end
         state_code.to_s
       end
+
+      def intake_class_from_state_code(state_code)
+        STATES_INFO[state_code][:intake_class]
+      end
+
+      def navigation_from_state_code(state_code)
+        STATES_INFO[state_code][:navigation]
+      end
+
+      def intake_classes
+        STATES_INFO.map { |_, state_info| state_info[:intake_class] }
+      end
+
+      def intake_class_names
+        intake_classes.map(&:name)
+      end
+
+      def submission_builder_from_intake_class(klass)
+        state_info = STATES_INFO.values.find { |s| s[:intake_class] == klass }
+        state_info[:submission_builder] if state_info.present?
+      end
```

Comment:
i wonder if we should define a method every time we want to get something via not-the-state-code. this for example could be obtained this way:
```
state_code = StateInformationService.state_code_from_intake_class(intake_class)
StateInformationService.submission_builder(state_code)
```
which is more verbose but doesn't result in a custom method. we could dynamically define a way to get any piece of information for a state given any other piece of information, but probably that's overkill...

also, i just realized i don't see this method used anywhere nor do i see a `submission_builder` defined in the STATES_INFO. is part of this PR missing?